aws/logs_monitoring/README.md

```diff
@@ -292,6 +292,8 @@ The Datadog Forwarder is signed by Datadog. If you would like to verify the inte
 - `IncludeAtMatch` - Only send logs matching the supplied regular expression and not excluded by
   ExcludeAtMatch. Note, using inefficient regular expression, such as `.*`, may slow down the Lambda function.
 
+**Note:** To debug these regular expressions against your logs, turn on [debug logs](#troubleshooting).
```

Comment:
- I think we should combine this note with the existing note about inefficient regular expression -- either duplicate this one into both parameters' description or extract that existing note out. 
- Can you add a quick/simple example?

Reply:
Per conversation in Slack, will revise the README but hold off on adding examples for now in the interest of getting this out.